src/docs/product/dashboards/widget-builder/index.mdx

```diff
@@ -68,7 +67,10 @@ Choosing "Errors and Transactions" allows you to query and aggregate error and t
 - Throughput by country
 - Users most affected by errors
 
+For performance widgets (based on transactions), [metrics-based](/product/sentry-basics/search/searchable-properties/#metrics-properties) data is the default. If you make a selection in configuring your widget that's not metrics-compatible, then your widget will use event-based data and a message saying this will be displayed.
```

Comment:
```suggestion
For performance widgets (based on transactions), [metric-backed](/product/sentry-basics/search/searchable-properties/#metrics-properties) data is the default. If you make a selection in configuring your widget that's not metrics-compatible, then your widget will use indexed events, or event-based data, and a message saying this will be displayed.

<Note>

Metric-backed data is in beta and is only available if your organization is participating in its limited release. Features in beta are still in-progress and may have bugs. We recognize the irony. If you’re interested in joining this limited release, please reach out at [abc@sentry.io](mailto:abc@sentry.io).

</Note>
```